packages/web/src/shared/fontClient.ts

```diff
@@ -0,0 +1,33 @@
+const FONT_API =
+  'https://www.googleapis.com/webfonts/v1/webfonts?key=' +
+  process.env.GOOGLE_FONT_API_KEY;
+
+class FontClient {
+  async getFonts() {
+    console.log(JSON.stringify(process.env));
+    console.log(process.env.API_URL);
+    console.log(process.env.GOOGLE_FONT_API_KEY);
```

Comment:
@arrocke, for some reason `process.env.GOOGLE_FONT_API_KEY` is always `undefined`. I defined it in `packages/web/.env.local`, but it doesn't seem to register. Is there anything special I need to do to get it to work?

Reply:
Lol. That function should be named `useLanguageQuery`. At my job, we have licenses as our top level org unit rather than languages. Probably not the last time I'll interchange them.

Reply:
It's really strange. I've tried that several times and it did not fix it. I also tried directly adding it to `.env` and restarting but that didn't work either.

Reply:
Ok, that worked. Not sure how `API_URL` works?

Reply:
API_URL works because of [this file](https://github.com/arrocke/gloss-translation/blob/main/packages/web/webpack.config.js) which configures webpack to replace references to API_URL with that string. Let's stick with the NX_ prefix so that we don't have modify our tooling for every env var